src/test/resources/retest/recheck/de.retest.web.it.ShowcaseIT/showcase-ChromeDriver.open.recheck/retest.xml

```diff
@@ -11350,10 +11408,6 @@
 					</attributes>
 				</containedElements>
 			</containedElements>
-			<screenshot>
-				<persistenceId>html</persistenceId>
```

Comment:
This can be removed too then: https://github.com/retest/recheck-web/blob/feature/travis-update-to-bionic/src/test/resources/retest/recheck/de.retest.web.it.ShowcaseIT/showcase-ChromeDriver.open.recheck/screenshot/html.png

Reply:
Removed in the current iteration.